src/ngraph/frontend/onnx_import/onnx.cpp

```diff
@@ -52,6 +52,19 @@ namespace ngraph
             } // namespace error
         }     // namespace detail
 
+        std::shared_ptr<Function> import_onnx_model(const onnx::ModelProto& model_proto)
```

Comment:
I wonder what will happen if there will be a missmatch between caller and our versions of onnx library

Reply:
It won't work, just like it currently doesn't if we choose to use "ngraph_onnx" (https://github.com/NervanaSystems/ngraph/pull/4530) and try to link against it in UEP.